modules/monitoring-disabling-grafana.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_grafana_{context}"]
+= Disabling the default Grafana deployment
+
+By default a read-only Grafana instance is deployed with a collection of dashboards displaying cluster metrics. This Grafana instance is not user-configurable.
```

Comment:
This Grafana instance
->
The Grafana instance

Reply:
Fixed.